docs/_newsfragments/2538.newandimproved.rst

```diff
@@ -0,0 +1 @@
+req.get_param_as_list() now supports a delimiter argument for splitting values.
```

Comment:
It would be nicer to actually link to the request method in question, probably the below could do (untested)
```rst
:meth:`falcon.Request.get_param_as_list`
```